doc/en/datetime.txt

```diff
@@ -402,6 +402,52 @@
 
 /* $DOC$
    $AUTHOR$
+      2017 Pete D. <pete_westg@yahoo.gr>
+   $TEMPLATE$
+      Function
+   $NAME$
+      hb_Week()
+   $CATEGORY$
+      API
+   $SUBCATEGORY$
+      Date/Time
+   $ONELINER$
+      Return the week number of year.
+   $SYNTAX$
+      hb_Week( <dDate>, [@<nYear>], [@<nDayOfWeek>] ) --> nWeek
+   $ARGUMENTS$
+      <dDate> Any valid date expression.
+
+		<nYear>, <nDayOfWeek> Optional parameters to obtain the year and day number of week.
```

Comment:
This needs to be indented one space right (otherwise will be picked wrongly by the HBDOC parse).